ocurrent-plugin/current_ocluster.ml

```diff
@@ -41,7 +45,7 @@ module Op = struct
     let commit_id_to_yojson x = `String (Git.Commit_id.hash x)
 
     type t = {
-      action : [`Docker of docker_build | `Obuilder of Cluster_api.Obuilder_job.Spec.t];
+      action : [`Docker of docker_build | `Obuilder of Cluster_api.Obuilder_job.Spec.t | `Custom of custom];
```

Comment:
```suggestion
      action : [`Docker of docker_build | `Obuilder of Cluster_api.Obuilder_job.Spec.t];
```

Reply:
Woops!! Thanks, I also removed the code that was needed because of this :))